core/arch/arm/plat-rpi3/main.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2016, Sequitur Labs Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
This license is different from the one we're usually using, is this intentional?